### DISCUSSION TITLE
UOSC cover other OSC things

### DISCUSSION THREAD

**Discussion by OmiChrome** [tomasklaen/uosc#498] [Q&A]
when something happen in mpv , like other script msg , UOSC flash up the topbar and cover the OSC 

![image](https://user-images.githubusercontent.com/108952834/230202344-9fd44fc5-f51d-46da-a506-50677c726e45.png)

here is the log video 
https://user-images.githubusercontent.com/108952834/230203531-5fa07177-0596-438b-9dfc-62d00f48d754.mp4

https://user-images.githubusercontent.com/108952834/230204027-81da6981-84e4-4637-a111-d41732e997db.mp4

info 
OS : win 11 
build : latest-git-builds ~ , mpv , ffmpeg 

my [uosc.conf](https://github.com/tomasklaen/uosc/files/11162858/uosc.conf.txt)

**Comment by tomasklaen** [comment]
Can't replicate. Needs steps to reproduce.

But the only time top bar flashes is when new file is loaded, or when you manually tell it to via `flash-ui` binding or `flash-elements` message.

**Reply by christoph-heinrich** [reply]
Depending on the configuration it can be problematic sometimes. e.g. `top_bar_persistency=paused,audio`

**Comment by christoph-heinrich** [comment]
We could try setting `osd-margin-y` like with do for `osc-margins`. Unfortunately the unit isn't the same, but based on the description of `sub-font-size` multiplying it with 720 should "just work". We should probably take the user specified option into account there by adding it.

**Reply by christoph-heinrich** [reply]
I've made an implementation you can try out #499.

**Reply by tomasklaen** [reply]
But the problem here, looking at the video, is that the top bar is flashing after he changes volume, and I'm not sure why that would happen apart of manual trigger via messages/bindings.

**Comment by hooke007** [comment]
It's all mpvconf's issue.
You were using a small size of OSD, which is easy to be covered by any elements. Vanilla osc could also reproduce the 'issue'. So the solution should be using the better osd setting.

**Comment by OmiChrome** [comment]
it answers everything   OSD setting must be changed

**Reply by OmiChrome** [reply]
# Hide UI when mpv autohides the cursor
`autohide=yes`
`mpv.conf` = `--osd-margin-y=50` # sweet spot for uosc ' osd

this worked well for now
